File: ZoneSentry/ClientApp/src/api/models/RealtyService.ts
/* istanbul ignore file */
/* tslint:disable */
/* eslint-disable */

import type { ApplicationUser } from './ApplicationUser';
import type { House } from './House';

export type RealtyService = {
    id?: number;
    name?: string | null;
    cost?: number;
    provider?: ApplicationUser;
    scope?: Array<House> | null;
};

File: ZoneSentry/Models/RealtyService.cs
namespace ZoneSentry.Models;

public class RealtyService
{
    public int Id { get; set; }
    public string Name { get; set; }
    public int Cost { get; set; }
    
    public ApplicationUser Provider { get; set; }
    
    public List<House> Scope { get; set; }
}

public class RealtyServiceDTO
{
    public int Id { get; set; }
    public string Name { get; set; }
    public int Cost { get; set; }
    
    public ApplicationUserDTO Provider { get; set; }
}

public class RealtyServiceUpdate
{
    public int Id { get; set; }
    public string Name { get; set; }
    public int Cost { get; set; }
}